server/broadcast_test.go

```diff
@@ -955,3 +1067,20 @@ func TestVerifier_HLSInsertion(t *testing.T) {
 	assert.Nil(err)
 	assert.Equal(baseURL+"/resp2", pl.uri)
 }
+
+func defaultTicketbatch() *pm.TicketBatch {
```

Comment:
Nit: "TicketBatch"

Reply:
fixed in 20340ceabc68452fb4c8914a568e965e1c5f5d30